.changeset/flat-trees-return.md

```diff
@@ -0,0 +1,5 @@
+---
+"app-builder-lib": minor
+---
+
+feat(nsis): terminate only processes running in installation folder
```

Comment:
Please change this to `patch` with `fix(nsis)`. This should partially fix a reported issue and also, the CD for some reason flags `minor` as a major semver bump, which we aren't ready to do yet.

Reply:
done